Dockerfile

```diff
@@ -0,0 +1,62 @@
+#***************************************************************************************
+# Copyright (c) 2025 Beijing Institute of Open Source Chip (BOSC)
+#
+# XiangShan is licensed under Mulan PSL v2.
+# You can use this software according to the terms and conditions of the Mulan PSL v2.
+# You may obtain a copy of Mulan PSL v2 at:
+#          http://license.coscl.org.cn/MulanPSL2
+#
+# THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+# EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+# MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+#
+# See the Mulan PSL v2 for more details.
+#***************************************************************************************
+
+FROM verilator/verilator:latest
+
+# override ENTRYPOINT in verilator image
+ENTRYPOINT [ "/bin/bash" ]
+ENV VERILATOR=/usr/local/bin/verilator-wrap.sh
+
+# explict set LC_ALL to C.UTF-8
+ENV LC_ALL=C.UTF-8
+
+#
+# Prepare envrionments for chisel/mill
+#
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive \
+    && apt-get install --no-install-recommends -y \
+			default-jdk \
+			python3 \
+			curl \
+			make \
```

Comment:
They have been installed on the xs-env docker. I think we should maintain the dependecies in one place

Reply:
fixed to use `ghcr.io/openxiangshan/xs-env:latest` as base image